### PR TITLE
Fix Switcheroo not removing statbar item note when no item received

### PR DESCRIPTION
**Pull request by asgdf**
[Example](http://replay.pokemonshowdown.com/gen7anythinggoes-542021819) on turn 7

### CODE REVIEW

File: data/moves.js [hunk 1]
```diff
@@ -16674,11 +16674,15 @@ exports.BattleMovedex = {
 			this.add('-activate', source, 'move: Trick', '[of] ' + target);
 			if (myItem) {
 				target.setItem(myItem);
-				this.add('-item', target, myItem, '[from] Trick');
+				this.add('-item', target, myItem, '[from] move: Switcheroo');
+			} else {
+				this.add('-enditem', target, yourItem, '[silent]');
```
Comment (Zarel) [inline data/moves.js]: You should still have `[from] move: Switcheroo`

Comment (asgdf) [inline data/moves.js]: Would that look like `'[silent] [from] move: Switcheroo'` or `'[silent]', '[from] move: Switcheroo'`?

**Comment by asgdf** [comment]
Ok, now it includes the source of the -enditem. I added that to Trick as well, while I was at it.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**